.github/workflows/main.yml

```
@@ -10,11 +10,14 @@ on:
       - 'LICENSE.md'
       - '.gitattributes'
       - '.gitignore'
-      - '/src/packs'
-      - '/src/module/helperCode'
+      - '/src/packs/**'
+      - '/src/module/helperCode/**'
+      - 'package-lock.json'
+      - 'package.json'
     
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)